### PR TITLE
Rename FixRelayedMoveBalanceFlag to FixRelayedBaseCostFlag + proper fix

### PR DESCRIPTION
**Pull request by sstanculeanu**
## Reasoning behind the pull request
- the fix for move balance should actually be fix for base cost
  
## Proposed changes
- updated the fix to also cover other inner txs than move balance

## Testing procedure
- will be verified on feat branch

## Pre-requisites

Based on the [Contributing Guidelines](https://github.com/multiversx/mx-chain-go/blob/master/.github/CONTRIBUTING.md#branches-management) the PR author and the reviewers must check the following requirements are met:
- was the PR targeted to the correct branch?
- if this is a larger feature that probably needs more than one PR, is there a `feat` branch created?
- if this is a `feat` branch merging, do all satellite projects have a proper tag inside `go.mod`?

### CODE REVIEW

File: process/transaction/shardProcess.go [hunk 1]
```diff
@@ -744,9 +743,7 @@ func (txProc *txProcessor) processInnerTx(
 	originalTxHash []byte,
 ) (*big.Int, vmcommon.ReturnCode, error) {
 
-	_, dstShardTxType := txProc.txTypeHandler.ComputeTransactionType(innerTx)
-	isMoveBalance := dstShardTxType == process.MoveBalance
-	txFee := txProc.computeTxFee(innerTx, isMoveBalance)
+	txFee := txProc.computeTxFee(innerTx)
```
Comment (sasurobert) [inline process/transaction/shardProcess.go:746]: rename to computeTxFeeForRelayedTx?

Comment (sstanculeanu) [reply process/transaction/shardProcess.go]: pushed

**Review by sasurobert — APPROVED** [review]

File: process/transaction/shardProcess.go [hunk 2]
```diff
@@ -865,10 +862,8 @@ func (txProc *txProcessor) processRelayedTx(
 func (txProc *txProcessor) computeRelayedTxFees(tx, userTx *transaction.Transaction) relayedFees {
 	relayerFee := txProc.economicsFee.ComputeMoveBalanceFee(tx)
 	totalFee := txProc.economicsFee.ComputeTxFee(tx)
-	_, dstShardTxType := txProc.txTypeHandler.ComputeTransactionType(userTx)
-	isMoveBalance := dstShardTxType == process.MoveBalance
-	if txProc.enableEpochsHandler.IsFlagEnabled(common.FixRelayedMoveBalanceFlag) && isMoveBalance {
```
Comment (raduchis) [inline process/transaction/shardProcess.go:870]: there was a check for && isMoveBalance. Is it ok that it's not used any more? 

New lines 1163 and 1170 on executeFailedRelayedUserTx still check for isMoveBalance.

Comment (sasurobert) [reply process/transaction/shardProcess.go:870]: Good catch. I think it needs to be deleted from there as well.

We should remove that isMoveBalance check from everywhere when we speak about relayed TX.  As you want to process the baseFee (gasPrice*(50K + len(data)*1500)) for every inner operation, as it happens on any tx.

Comment (sstanculeanu) [reply process/transaction/shardProcess.go:870]: right, pushed

Comment (raduchis) [reply process/transaction/shardProcess.go:870]: ok!

**Review by raduchis — APPROVED** [review]

**Review by sasurobert — APPROVED** [review]

**Pull request merged**